run_dir/static/js/running_notes.js

```diff
@@ -143,34 +144,55 @@ function filter_running_notes(search){
         }
     });
 }
-//Filter notes by Category
+//Filter notes
 $('#rn_search').keyup(function() {
     var search=$('#rn_search').val();
     filter_running_notes(search);
 });
-$('.btnCatFilter').click(function() {
-    $('.btnCatFilter').each(function() {$(this).removeClass("glow")});
-    $(this).addClass("glow")
-    var search=$(this).text();
-    if (search == 'All'){
-        search='';
-    }
-    filter_running_notes(search);
+
+$(document).ready(function() {
+    $('#rn_category').change(function() {
+        var search=$('#rn_category :selected').text();
+        if (search == 'All'){
+            search='';
+        }
+        filter_running_notes(search);
+    });
 });
+
+// Update the category buttons
+$('.btnCatFilter').click(function(){
+    var was_selected = $(this).hasClass('active');
+    $('.btnCatFilter').removeClass('active');
+    if(!was_selected){ $(this).addClass('active'); }
+    });
+
+//Remove hover text from clicked button
+$(document).ready(function(){
+       $('[data-toggle="tooltip"]').click(function () {
+          $('[data-toggle="tooltip"]').tooltip("hide");
+       });
+})
+
 // Preview running notes
 $('#new_note_text').keyup(preview_running_notes);
```

Comment:
Add `()` to preview function call (I'm surprised it worked without these?)
```suggestion
$('#new_note_text').keyup(preview_running_notes());
```

Reply:
Yeah, I don't know why this isn't working, but it isn't.....oh well

Reply:
This is weeeeird but fine - use whatever works!